src/api/request-validations.ts

```diff
@@ -85,3 +85,10 @@ export const validateXMLBodyEquals = async (request: RequestExt, expectedBody: s
     throw new ValidationError(BODY_NOT_EQUAL_ERROR_MESSAGE, expectedParsedBody, actualParsedBody);
   }
 };
+
+export const validateHeader = (request: RequestExt, header_name: string, expected: string) => {
+  const actual = request.headers[header_name];
+  if (actual != expected) {
```

Comment:
realize there is no linting rule for that, so I'll one but you should pretty much always use triple `=` in javascript
`===` or `!==` . Using `==` has unexpected consequences